sources/packages/backend/libs/services/src/confirmation-of-enrollment/models/confirmation-of-enrollment.models.ts

```diff
@@ -0,0 +1,35 @@
+import { DisbursementValue, EducationProgramOffering } from "@sims/sims-db";
+
+/**
+ * Award information needed for the max tuition remittance calculation.
+ */
+export type Award = Pick<
```

Comment:
Why using Pick?

Reply:
It is to create a subset from the DB entity model to express precisely what this specific method requires to be executed.
Besides that, if the entity model ever changes this will also follow whatever type is changed on the Pick type.
For instance, the offering entity model has way more properties but we are using only two, for now. We can create a new object with only two properties or just reuse the entity model one.
I am not always looking forward to using it (Pick) but this calculation, as far as we know, will always be receiving the DB entities models as parameters.

Reply:
ok. Thanks for the explanation!